assets/js/components/user-input/util/constants.js

```diff
@@ -40,6 +40,51 @@ export const USER_INPUT_MAX_ANSWERS = {
 export const USER_INPUT_CURRENTLY_EDITING_KEY =
 	'googlesitekit-user-input-currently-editing';
 
+export const FORM_USER_INPUT_QUESTION_NUMBER = 'user_input_question_number';
+
+/**
+ * Gets available questions for user input settings.
+ *
+ * @since n.e.x.t
+ * @private
+ *
+ * @return {Object} Questions object.
+ */
+export function getUserInputQuestions() {
+	return {
+		1: {
```

Comment:
It might be slightly simpler to use an array with objects, but I guess you used objects with numerical keys to avoid incrementing indexes for +1 ?